src/util/spiderfyMarkers.js

```diff
@@ -0,0 +1,16 @@
+omsUtil = function (map, options) {
+    var oms = new OverlappingMarkerSpiderfier(map, options);
+
+    var popup = new L.Popup();
+    oms.addListener('click', function(marker) {
+        popup.setContent(marker._popup._content);
```

Comment:
Are there any leaflet functions to get content from marker?

Reply:
I will check the documentation and give you an update.

Reply:
I have refactored the code to make use of the leaflet method to get popup information in my commit 6ecf40f. Thanks for suggesting it!